backend/workflow_manager/workflow_v2/views.py

```diff
@@ -116,16 +114,7 @@ def perform_create(self, serializer: WorkflowSerializer) -> Workflow:
                 is_active=True,
             )
             WorkflowEndpointUtils.create_endpoints_for_workflow(workflow)
-
-            # Enable GCS configurations to create GCS while creating a workflow
-            if (
-                settings.GOOGLE_STORAGE_ACCESS_KEY_ID
-                and settings.UNSTRACT_FREE_STORAGE_BUCKET_NAME
-            ):
-                ConnectorInstanceHelper.create_default_gcs_connector(
-                    workflow, self.request.user
-                )
```

Comment:
@muhammad-ali-e I don't think we use this but we might have still created it previously - do you think we'll need a migration to remove those? It might cause regressions if we have any future logic that deviates from what we've defined in it

Reply:
I didn’t encounter any issues when removing it, and I’ve checked with some existing data as well. If any regressions occur, we can address them by adding this back. We should only run the migrations after thoroughly testing and confirming that there are no issues or regressions.

Reply:
I think not using it. Want to check with @johnyrahul

Reply:
@chandrasekharan-zipstack We are not using any default connector now.